verification/Spreadsheets/FiatToken - UnitTestCompleteness.csv

```diff
@@ -0,0 +1,36 @@
+File,Function,Modifier,Simple Test,ABI Test,Pausable,Code: paused=true,Blacklistable,blacklisted[msg.sender]=true,blacklisted[address]=true,Only 1 User,Wrong msg.sender

```

Comment:
the csv test verification is disabled on master, but leaving these here in case we want to re-enable it

Reply:
removed per offline discussion w/ @eztierney